crates/bevy_macro_utils/src/lib.rs

```diff
@@ -37,18 +37,15 @@ const BEVY_INTERNAL: &str = "bevy_internal";
 
 impl BevyManifest {
     pub fn maybe_get_path(&self, name: &str) -> Option<syn::Path> {
-        fn dep_package(dep: &Value) -> Option<&str> {
+        fn dep_package(dep: &Item) -> Option<&str> {
             if dep.as_str().is_some() {
                 None
             } else {
-                dep.as_table()
-                    .unwrap()
-                    .get("package")
-                    .map(|name| name.as_str().unwrap())
+                dep.get("package").map(|name| name.as_str().expect("2"))
```

Comment:
Why change the `unwrap()` to an `expect("2")`?

Reply:
Ah shoot, leftover from debugging. Reverting this.